### ISSUE TITLE
tabs in multi-line literal strings yield error

### ISSUE THREAD

**Issue by rottaran** [avakar/pytoml#46]
The TOML specification states that literal-strings are interpreted as-is. Hence, tab characters in literal strings should be possible. Our TOML files contain partial makefile rules and as such require tab characters.

> Multi-line literal strings are surrounded by three single quotes on each side and allow newlines. Like literal strings, there is no escaping whatsoever. A newline immediately following the opening delimiter will be trimmed. All other content between the delimiters is interpreted as-is without modification.

However, the decoding fails with the usual generic error. It works in pytoml version 0.1.2

```
File ".../pytoml/parser.py", line 10, in load
    return loads(fin.read(), translate=translate, object_pairs_hook=object_pairs_hook, filename=getattr(fin, 'name', repr(fin)))
  File ".../pytoml/parser.py", line 23, in loads
    ast = _p_toml(src, object_pairs_hook=object_pairs_hook)
  File ".../pytoml/parser.py", line 352, in _p_toml
    s.expect_eof()
  File ".../pytoml/parser.py", line 124, in expect_eof
    return self._expect(self.consume_eof())
  File ".../pytoml/parser.py", line 164, in _expect
    raise TomlError('msg', self._pos[0], self._pos[1], self._filename)
pytoml.core.TomlError: somefilename(14, 5): msg
```

**Comment by avakar** [comment]
Thanks for the issue, I'll try to have a look at it later today.

**Comment by rottaran** [comment]
https://github.com/avakar/pytoml/blob/cb92445ca769b3966b3976cd69f5140761f15843/pytoml/parser.py#L227
```
_litstr_ml_re = re.compile(r"(?:(?:|'|'')(?:[^'\000-\011\013-\037]))*")
```

maybe excluding all control characters is too much there.

**Comment by avakar** [comment]
Fixed in 0.1.19, run `pip install -U pytoml` to get the fix.
